packages/headless/src/case-assist.index.ts

```diff
@@ -89,6 +89,9 @@ export {buildCaseAssistInteractiveResult as buildInteractiveResult} from './cont
 export {buildDocumentSuggestionList as buildDocumentSuggestion} from './controllers/document-suggestion-list/headless-document-suggestion-list';
 
 export type {
+  InteractiveResultCore,
```

Comment:
🤔 Why is this necessary to export ? 

This should be internal only.

Reply:
This interface is necessary to export because the InteractiveResult extends from it. It is already exported by default, this addition just makes it explicit for the doc-parser. It will result in a doc page looking like the product listing facet one. Here in the index file, the Core interfaces are being exported.

<img width="994" alt="image" src="https://github.com/coveo/ui-kit/assets/78121423/23f98fba-843c-4b39-9212-c41b2bf62d72">

And in the documentation, they are not showed, the doc-parser simply builds the documentation using information from the Facet controller and the FacetCore one.

https://docs.coveo.com/en/headless/latest/reference/product-listing/controllers/facet/